src/containers/PageBuilder/Primitives/List/List.module.css

```diff
@@ -14,9 +14,16 @@
       list-style-type: square;
     }
   }
+
+  & + * {
+    margin-top: 24px;
+  }
```

Comment:
What does this mean?
A list (ul & ol) should not be used unless there's a list-item (li) inside.
Did you find a bug?

Reply:
@Gnito it means that if there's an element after the List component, the element gets a margin-top and that determines the white-space between the List component and the element. If there's no element after the List component, there won't be a white-space as it's not necessary.